.circleci/config.yml

```diff
@@ -360,9 +376,15 @@ workflows:
       - test_rule_help_version:
           requires:
             - build_unix
+      - test_jsdom:
+          requires:
+            - build_unix
       - test_node:
           requires:
             - build_unix
+          matrix:
+            parameters:
+              node-version: ['6', '8', '10', '12', '14', '16', '18', '20']
```

Comment:
Do we really need all of these?

Reply:
Yes. This is every version of node we claim to support so we should have a test for it. They only need 30s of time each and run in parallel so don't make the build any longer.